compiler/include/initializerResolution.h

```diff
@@ -27,13 +27,17 @@
 // handled, as we want to have the this argument for the type in the initializer
 // argument list, and want to utilize Phase 1 of the initializer body to
 // determine the generic instantiation of it.
+class AggregateType;
```

Comment:
I don't think callInfo.h needs to be included in this header file

I also don't think it needs BlockStmt, DefExpr, Expr, or Symbol.